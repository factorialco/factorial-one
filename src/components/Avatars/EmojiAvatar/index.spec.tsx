import { render } from "@testing-library/react-native";
import React from "react";
import { EmojiAvatar } from "./";

describe("EmojiAvatar", () => {
  it("Snapshot - different sizes text emoji and diferent size", () => {
    const sizes = ["sm", "md", "lg"] as const;

    sizes.forEach((size) => {
      const { toJSON } = render(<EmojiAvatar emoji="test" size={size} />);

      expect(toJSON()).toMatchSnapshot();
    });
  });
});
