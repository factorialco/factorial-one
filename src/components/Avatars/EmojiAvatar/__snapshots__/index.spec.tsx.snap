// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmojiAvatar Snapshot - different sizes text emoji and diferent size 1`] = `
<View
  className="w-6 h-6 rounded-sm flex aspect-square items-center justify-center border border-solid border-f1-border-secondary bg-f1-background dark:bg-f1-background-inverse-secondary"
>
  <Text
    className="text-xs"
  >
    test
  </Text>
</View>
`;

exports[`EmojiAvatar Snapshot - different sizes text emoji and diferent size 2`] = `
<View
  className="w-9 h-9 rounded-md flex aspect-square items-center justify-center border border-solid border-f1-border-secondary bg-f1-background dark:bg-f1-background-inverse-secondary"
>
  <Text
    className="text-sm"
  >
    test
  </Text>
</View>
`;

exports[`EmojiAvatar Snapshot - different sizes text emoji and diferent size 3`] = `
<View
  className="w-10 h-10 rounded-lg flex aspect-square items-center justify-center border border-solid border-f1-border-secondary bg-f1-background dark:bg-f1-background-inverse-secondary"
>
  <Text
    className="text-md"
  >
    test
  </Text>
</View>
`;
